polaris-core/src/main/resources/schemas/policies/system/orphan-file-removal/2025-02-03.json

```diff
@@ -0,0 +1,47 @@
+{
+  "license": "Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)",
+  "$id": "https://polaris.apache.org/schemas/policies/system/orphan-file-removal/2025-02-03.json",
+  "title": "Orphan File Removal Policy",
+  "description": "Inheritable Polaris policy schema for Iceberg table orphan file removal.",
```

Comment:
1. What does `Inheritable` mean here?
2. `Polaris` seems redundant, all of these are going to be Polaris schemas right?

Reply:
A inheritable policy means it can be applied to the under layer entities. For example, all tables under a namespace get the policies if it is assigned to the namespace. 
I'm OK to remove it or keep it. Keeping it provides a complete view for anyone who read the schema, but without too much context of Polaris.

Reply:
Are all policies inheritable?

Reply:
No. For example, column masking policy are not inheritable, which makes more sense, and also that's what most engines do.